playground/backend/internal/utils/precompiled_objects_utils.go

```diff
@@ -37,3 +46,37 @@ func PutPrecompiledObjectsToCategory(categoryName string, precompiledObjects *cl
 	}
 	sdkCategory.Categories = append(sdkCategory.Categories, &category)
 }
+
+// GetPrecompiledObjectsCatalogFromCache returns the precompiled objects catalog from the cache
+func GetPrecompiledObjectsCatalogFromCache(ctx context.Context, cacheService cache.Cache) ([]*pb.Categories, error) {
```

Comment:
can we make this name shorter?
GetPrecompiledObjectsFromCache

Reply:
This one became redundant and `GetPrecompiledObjectsCatalogFromStorage` I changed to `GetCatalogFromStorage`